app/src/main/java/ch/sdp/vibester/user/UserProfileAdapter.kt

```diff
@@ -76,11 +76,12 @@ class UserProfileAdapter constructor(
         /**
          * @param user with all the parameters
          */
-        fun bind(user: User) {
+        fun bind(user: User, position: Int) {
             itemView.findViewById<TextView>(R.id.search_user_username).text = user.username
+            if(position %2 == 0) itemView.setBackgroundColor(Color.parseColor("#FFFDF0D7"));
```

Comment:
```suggestion
            if(position %2 == 0) itemView.setBackgroundColor(Color.parseColor("#FFFDF0D7"))
```

Reply:
Why not adding it to the color.xml file ?

Reply:
they are there, but ican parse them correctly since this file is not an activity. do you have an idea how?

Reply:
remove colons!

Reply:
For your question on telegram, you should use that to access the colors.xml:
`itemView.setBackgroundColor(itemView.resources.getColor(R.color.black))`

Reply:
i see, fixed!